shell/platform/android/ndk_helpers.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_ANDROID_NDK_HELPERS_H_
+#define FLUTTER_SHELL_PLATFORM_ANDROID_NDK_HELPERS_H_
+
+#include "flutter/fml/platform/android/jni_util.h"
+
+#include "flutter/impeller/renderer/backend/gles/gles.h"
+
+#include <android/hardware_buffer.h>
+
+namespace flutter {
+
+// A collection of NDK functions that are available depending on the version of
+// the Android SDK we are linked with at runtime.
+class NDKHelpers {
+ public:
+  // REQUIRED: Must be called before any methods below are.
+  static void Init();
```

Comment:
Can this just be called implicitly using `std::call_once` internally?

Reply:
Done.